catalogs/catalog-jdbc-postgresql/src/test/java/com/datastrato/gravitino/catalog/postgresql/integration/test/CatalogPostgreSqlIT.java

```diff
@@ -156,16 +156,18 @@ private void createCatalog() {
     Map<String, String> catalogProperties = Maps.newHashMap();
 
     try {
-      String jdbcUrl = POSTGRESQL_CONTAINER.getJdbcUrl();
+      String jdbcUrl = POSTGRESQL_CONTAINER.getJdbcUrl(TEST_DB_NAME);
       String database = new URI(jdbcUrl.substring(jdbcUrl.lastIndexOf("/") + 1)).getPath();
       catalogProperties.put(
-          JdbcConfig.JDBC_DRIVER.getKey(), POSTGRESQL_CONTAINER.getDriverClassName());
+          JdbcConfig.JDBC_DRIVER.getKey(), POSTGRESQL_CONTAINER.getDriverClassName(TEST_DB_NAME));
       catalogProperties.put(JdbcConfig.JDBC_URL.getKey(), jdbcUrl);
       catalogProperties.put(JdbcConfig.JDBC_DATABASE.getKey(), database);
       catalogProperties.put(JdbcConfig.USERNAME.getKey(), POSTGRESQL_CONTAINER.getUsername());
       catalogProperties.put(JdbcConfig.PASSWORD.getKey(), POSTGRESQL_CONTAINER.getPassword());
     } catch (URISyntaxException e) {
       throw new RuntimeException(e);
+    } catch (SQLException e) {
+      throw new RuntimeException(e);
```

Comment:
This 'catch' blocks can be collapsed

Reply:
Okay, I throws exception in method signature.

```
 public void executeQuery(String sql) throws SQLException {
    Statement statement = connection.createStatement();
    statement.execute(sql);
  }
```

Reply:
I mean you can change it like:
```java
catch (URISyntaxException | SQLException e) {
      throw new RuntimeException(e);
    }
```

Reply:
Oh, since this belongs to the test module, we can just throw an exception in the method signature.

Reply:
> Oh, since this belongs to the test module, we can just throw an exception in the method signature.